forge-gui/res/cardsfolder/upcoming/archangel_of_wrath.txt

```diff
@@ -0,0 +1,12 @@
+Name:Archangel of Wrath
+ManaCost:2 W W
+Types:Creature Angel
+PT:3/4
+K:Kicker:B:R
+K:Flying
+K:Lifelink
+T:Mode$ ChangesZone | ValidCard$ Card.Self+kicked | Origin$ Any | Destination$ Battlefield | Execute$ TrigDamage | TriggerDescription$ When CARDNAME enters the battlefield, if it was kicked, it deals 2 damage to any target.
+T:Mode$ ChangesZone | ValidCard$ Card.Self+kicked | Origin$ Any | Destination$ Battlefield | CheckSVar$ X | SVarCompare$ EQ2 | Execute$ TrigDamage | TriggerDescription$ When CARDNAME enters the battlefield, if it was kicked twice, it deals 2 damage to any target.
```

Comment:
for this i asked the rules manager about "kicked twice" or stuff that might increase the Kicker amount like "Sticker Kicker" but they didn't answer me yet

Reply:
Okay, I'll wait on an update for now. I can update this script in the future and revisit Temporal Firestorm in the same way if necessary.